partest-suite/src/test/resources/scala/tools/partest/scalajs/2.11.6/BlacklistedTests.txt

```diff
@@ -31,6 +31,9 @@ neg/t6289
 # RUN
 #
 
+# Too slow, really too slow
+run/bridges.scala
+
```

Comment:
Oh there is? I was looking for "slow" and things like that, didn't find anything ^^ I'll move it.